WordPressVIPMinimum/Sniffs/AbstractVariableRestrictionsSniff.php

```diff
@@ -9,7 +9,7 @@
 
 namespace WordPressVIPMinimum\Sniffs;
 
-use WordPressVIPMinimum\Sniffs\Sniff;
```

Comment:
The `WordPressVIPMinimum\Sniffs\Sniff` class is still being used....

Reply:
It is, but the `namespace` of this Abstract is the same as `Sniff`, so the `use` statement is not needed.